pkg/kubectl/cmd/cmd_test.go

```diff
@@ -354,8 +360,8 @@ func Example_printPodWithWideFormat() {
 		fmt.Printf("Unexpected error: %v", err)
 	}
 	// Output:
-	// NAME      READY     STATUS     RESTARTS   AGE       IP         NODE
-	// test1     1/2       podPhase   6          10y       10.1.1.3   kubernetes-node-abcd
+	// NAME      READY     STATUS     RESTARTS   AGE       IP         NODE                   REFERENCE
+	// test1     1/2       podPhase   6          10y       10.1.1.3   kubernetes-node-abcd   ReplicationController/my-rc
```

Comment:
How will this new column help? We may not want to make the output of `kubectl get pods` too long as this. The `REFERENCE` info can be found as `Created By` in `kubectl describe pod`.

If we do need this column, I'd prefer to display the controller in short name.